test/roundtrip/fold-try-delegate.txt

```diff
@@ -1,28 +1,31 @@
 ;;; TOOL: run-roundtrip
 ;;; ARGS: --stdout --fold-exprs --enable-exceptions --debug-names
 (module
-  (func (result i32)
-    try (result i32)
+  (func
+    try
       try
         nop
-      delegate 1
-      i32.const 7
+      delegate 0
+      try
+        nop
+      delegate 0
```

Comment:
This test has changed.. Which fix is this relevant relevant to?

Reply:
This change should now only be in the other PR #1685 for printing. The reason this test changes, btw, is just to align it with the new test `roundtrip/try-delegate.txt` in having multiple adjacent try-delegate blocks to exercise the relevant code path.